data/human/weapons.txt

```diff
@@ -556,8 +556,8 @@ outfit "Particle Cannon"
 		"firing force" 10
 		"firing heat" 120
 		"hit force" 80
-		"shield damage" 65
-		"hull damage" 62
+		"shield damage" 32.5
```

Comment:
I'm wondering if it might not be better for this one to be half as fast instead of hit half as hard?

Make it the big bad weapon in Human space that still holds up even in and a bit beyond Hai, if you can hit with it, but give it half as many chances to hit in the first place.

This would also further differentiate it from the somewhat competing Proton as well.

I could see this also being better as a follow up refinement, but since it still maintains the change in DPS ratio, I think it's also still appropriate here.

Reply:
You mean fire half as fast, and not the projectile going half as fast, right?

Reply:
Yea

Reply:
Wouldn't that remove it's main advantage being that it staggers making jumping hard?

Reply:
It's the proton canon you gotta use for that iirc

Reply:
Particle cannon is just as good, but it can be easily tested to see if it still stunlocks

Reply:
> Wouldn't that remove it's main advantage being that it staggers making jumping hard?

I am okay with it losing best stunlock spot when it is keeping the higher current per shot damage. That it is highest damage and range -and- best stun option in human space all at once is probably overpowered anyway.

Reply:
If you want to make the particle cannon deal twice the damage per hit and fire half as fast, that would be a separate change, which could be made with or without this change.

Reply:
Twice the damage and half as fast was just because the nerf was 50% at that point, meaning it would keep the per shot damage but still suffer the same DPS drop as other weapons.

Now that this has grown beyond just species' weapons, though, it is indeed much less appropriate here vary how we apply the overall changes for any particular weapon.